config/manager/manager.yaml

```diff
@@ -2,25 +2,25 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
-  name: system
+    control-plane: ssp-controller-manager
```

Comment:
this should be `ssp-operator` as well? (looking at the same label on the resources below)

Reply:
corrected